lab_2_keywords_cooccurrence/main.py

```diff
@@ -17,7 +26,12 @@ def extract_phrases(text: str) -> Optional[Sequence[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not(isinstance(text, str) and text):
+        return None
+    for punc in punctuation:
+        text = text.replace(punc, '.')
+        phrases = text.split('.')
+    return phrases
```

Comment:
I believe, this realization returns a list of tokens that have a lot of empty strings. Also, some of the tokens are going to include spaces at the beginning or/and at the end.